src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -785,10 +785,15 @@
                                   Header="{x:Static p:Resources.DynamoViewDebugMenuShowDebugAST}"
                                   IsChecked="{Binding ShowDebugASTs}"></MenuItem>
                         <MenuItem Focusable="False"
-                                  Name="ExperimentalSearch"
+                                  Name="TruncateSearchResults"
```

Comment:
@ColinDayOrg FYI I renamed your menu item

Reply:
Good catch, thanks. I am assuming that the "TruncateSearchResults" menu has been checked to test if it still works as expected after this change?

Reply:
I just did a quick test to verify that the menu still works in the current master codebase after this specific change, so it seems ok to me.